src/repository.ts

```diff
@@ -390,7 +444,7 @@ export class SinglePackageRepo extends Repository {
   protected async init(): Promise<void> {
     this.logger = await Logger.child(this.constructor.name);
     this.package = await Package.create();
-
+    this.shouldBePublished = await this.isReleasable(this.package);
```

Comment:
So here is a question. If we force update the version in `package.json` without a `fit` or `feat` commit message, do we still want to release it? For example, maybe in `salesforcedx` we have a commit message on Thursday as `chore: release 50.17.0` where we hardcode the version in that commit. In that case we want to release. It feels weird to have the commit be `feat: release 50.17.0`.

Thoughts?

Reply:
Maybe to stay consistent we do use `feat` to kick off the release... just pointing it out.

Reply:
That's a good point. I'll update it to ignore this check if the version bump has been hardcoded in the package.json